acl/pom.xml

```diff
@@ -50,5 +49,17 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-junit4</artifactId>
+            <version>1.7.1</version>
```

Comment:
The project has already use mockito-core. It will be better to keep consistent with other modules.

Reply:
ok，Has been modified